example/src/App.js

```diff
@@ -410,6 +411,19 @@ const App = () => {
         {JSON.stringify(inputCheckboxValue)}
       </Section>
 
+      <Section title="Input.Radio">
+        <h1>Input.Radio</h1>
+        <Input
+          onChange={(e) =>
+            console.log(e.target.checked) ||
```

Comment:
This isn't gonna hurt anything, but you don't need to leave a console log in here. If I did it for checkbox, it was a mistake.

Reply:
going to remove this from checkbox as well